### PR TITLE
More docs fixes/updates

### PR DESCRIPTION
**Pull request by PeadarOhAodha**
Raising this PR as a catch all for fixes and tweaks as I work through. 

- [x] Fix quickstart tutorial placement
- [x] Concepts > Datasets
- [x] Concepts > Evaluators
- [x] Various tweaks like removing reference to project
- [x] Removal of redundant guides
- [x] Cleanup deployment options 
- [x] Create Dataset guide

### CODE REVIEW

**Comment by jordn** [comment]
`defaults write com.apple.screencapture disable-shadow -bool true ; killall SystemUIServer` if you want to get rid of the shadow on screenshots.

(btw 1MB images isn't great, but not something we should care about rn)

**Review by jordn — APPROVED** [review]

**Comment by PeadarOhAodha** [comment]
> `defaults write com.apple.screencapture disable-shadow -bool true ; killall SystemUIServer` if you want to get rid of the shadow on screenshots.
> 
> (btw 1MB images isn't great, but not something we should care about rn)

i've switched to using Safari as well, much cleaner screenshots

File: fern/versions/v5.yml [hunk 1]
```diff
@@ -80,9 +80,9 @@ navigation:
               - page: Call a Prompt
                 path: ../v5-pages/development/guides/call-prompt.mdx
                 slug: call-prompt
-              - page: Proxying model calls
+              - page: Proxy model calls
                 path: ../v5-pages/development/guides/call-llm.mdx
-              - page: Logging LLM calls
```
Comment (jordn) [inline fern/versions/v5.yml:85]: disagree with the title change

**Comment by jordn** [comment]
note to go back and fix any broken URL https://www.notion.so/humanloop/Shared-links-to-docs-af0f259e90c445739b33e88fe2305270?pvs=6&utm_content=af0f259e-90c4-4573-9b33-e88fe2305270&utm_campaign=T0124SFJG1X

File: fern/versions/v5.yml (hunk 1 above)
Comment (PeadarOhAodha) [reply fern/versions/v5.yml:85]: pointed out in notion, but i they should be consistent, the diff conventions makes things messy imo. Don't feel too strongly which way

**Pull request merged**